src/linpde_gp/linfuncops/_select_output.py

```diff
@@ -12,6 +12,8 @@ def __init__(
         input_shapes: tuple[ShapeLike, ShapeLike],
         idx,
     ) -> None:
+        if isinstance(idx, tuple) and len(idx) == 1:
+            idx = idx[0]
```

Comment:
`SelectOutput`'s `idx` is used to index into a list of stacked functions or stacked covariance functions, and list indices must be integers. But in `LinearDifferentialOperator.to_sum`, we use the codomain indices - which are tuples - to construct `SelectOutput` instances. I thought about just doing the conversion there, but I think `SelectOutput` should be responsible for this, since further down the line we might also add support for outputs of higher dimension (e.g. matrices).

Reply:
So really, right now it's kind of a hack and my "idea" was that in the future `SelectOutput` would just handle tuple indices more elegantly.

Reply:
As discussed offline, we could internally represent stacked functions and stacked covariance functions with object arrays to mitigate this issue.